.vscode/settings.json

```diff
@@ -11,7 +11,7 @@
   "[python]": {
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-      "source.organizeImports": true
+      "source.organizeImports": "explicit"
```

Comment:
Seems like this got added by accident, want to strip it from this PR?

(...and probably, we do still want format on save for imports too?)